tt_metal/hw/firmware/src/brisc.cc

```diff
@@ -354,43 +357,69 @@ int main() {
     // Wait for ncrisc to halt
     wait_for_ncrisc_to_halt();
 
-    mailboxes->launch.go.run = RUN_MSG_DONE;
+    mailboxes->go_message.signal = RUN_MSG_DONE;
 
     while (1) {
         init_sync_registers();
         reset_ncrisc_with_iram();
 
         WAYPOINT("GW");
-        while (mailboxes->launch.go.run != RUN_MSG_GO);
+
+        while (mailboxes->go_message.signal != RUN_MSG_GO) {
```

Comment:
reading this into a local here and testing the local below will save some L1 BW

Reply:
Done